vendor/shims/ecpo-jquery.js

```diff
@@ -1,12 +1,21 @@
-// Define an amd '-jquery' shim which is used by ember-cli-page-object internally
+// Define an amd module called "-jquery" 
+// which exposes jQuery bundeled with ember-cli-page-object.
+//
+// This mode is used when we deal with `jquery`-less apps.
```

Comment:
Could we include in this comment something about the relationship between this shim and `vendor/ecpo-jquery/dist/jquery.js`?

Reply:
> vendor/ecpo-jquery/dist/jquery.js

Do you want this path to be specified here? I'm not opposed but I think it can easily become outdated if we change the path in the `index.js`. 
I'm open for suggestions.

Reply:
Oh yah, no need to specify the path...just would be helpful to add some context about this shim is used. For example, it expects that we have already imported our own version of jQuery, right?

Reply:
ok, seems reasonable